docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1842,6 +1848,31 @@ Comma-separated list of attributes to include in all new spans, such as `key1:va
 
 Can be set with the environment variable `OTEL_RESOURCE_ATTRIBUTES` (use `=` instead of `:` with the environment variable).
 
+### sampler_type
+
+Default value is `const`.
+
+Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`.
+
+### sampler_param
+
+Default value is `1`.
+
+Sampler configuration parameter. Depending on the value of `sampler_type`, it can be `0`, `1`, or a decimal value in between.
+
+- For `const` sampler, `0` or `1` to always `false`/`true` respectively
+- For `probabilistic` sampler, a probability between `0.0` and `1.0`
+- For `rateLimiting` sampler, the number of spans per second
+- For `remote` sampler, param is the same as for `probabilistic`
+  and indicates the initial sampling rate before the actual one
```

Comment:
```suggestion
 the initial sampling rate before a sample is received from the sampling server.
```

Reply:
This no longer makes sense as samples are not received from the sampling server...

Also, a multi-line suggestion would be _much_ easier to reason through 😅 I'm going to start a new one with a counter-proposal.

Reply:
https://github.com/grafana/grafana/pull/73587/files#r1320166327